precompiles/ArbOwner.go

```diff
@@ -44,6 +44,11 @@ func (con ArbOwner) RemoveChainOwner(c ctx, evm mech, addr addr) error {
 	return c.State.ChainOwners().Remove(addr, c.State.ArbOSVersion())
 }
 
+// RectifyChainOwner checks if the account is a chain owner
+func (con ArbOwner) RectifyChainOwner(c ctx, evm mech, addr addr) error {
```

Comment:
ArbOwner is permissioned and only accessible by chain owners. In theory I think we can make this method permissionless right? If so we could move it to ArbOwnerPublic

Reply:
Also, let's emit a log when this is called for debuggability. You can look at ArbRetryableTx's Keepalive method which emits a LifetimeExtended log for an example of how to do this.